Framework/NexusGeometry/src/NexusGeometrySave.cpp

```diff
@@ -650,7 +651,7 @@ class NexusGeometrySaveImpl {
   enum class Mode { Trunc, Append };
 
 private:
-  const Mode m_mode;
```

Comment:
Why has this been unmarked as const? We only set it during initialisation currently?

Reply:
was caused by a previosu merge, fixed